RTQA/CouchDB_Code/db_schema.py

```diff
@@ -0,0 +1,49 @@
+import json
+from safety_db import INSECURE, INSECURE_FULL
+
+changeset_schema = {
```

Comment:
It looks like this script is just used to generate the schema files? If so, it doesn't need to be included in this repo, as it violates [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) --- i.e., could create a situation where someone might change the schema in this script but then forgets to update the XYZ_schema.json file, leading to confusion.